libobs/obs-cocoa.m

```diff
@@ -156,7 +163,7 @@ static void log_available_memory(void)
 static void log_os(void)
 {
 	NSProcessInfo *pi = [NSProcessInfo processInfo];
-	blog(LOG_INFO, "OS Name: Mac OS X");
+	blog(LOG_INFO, "OS Name: macOS");
```

Comment:
this change is unrelated, but I guess it's fine since by now most OS commands also report macOS